src/Files.App/Data/Contexts/MultiPanes/IMultiPanesContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Contexts
+{
+	/// <summary>
+	/// Represents context for <see cref="MainPanesPage"/>, which manages multiple panes.
+	/// </summary>
+	public interface IMultiPanesContext
+	{
+		/// <summary>
+		/// Gets invoked when active pane is changing.
+		/// </summary>
+		event EventHandler? ActivePane_Changing;
+
+		/// <summary>
+		/// Gets invoked when active pane is changed.
+		/// </summary>
+		event EventHandler? ActivePane_Changed;
```

Comment:
Does the inclusion of underscores in event names follow naming conventions? I think they should be named in pascal case without underscores.

Reply:
Sure.

Reply:
Done